proto/panacea/datapool/v2/genesis.proto

```diff
@@ -25,4 +24,10 @@ message Params {
   ];
   uint64 data_pool_code_id = 2;
   string data_pool_nft_contract_address = 3;
+}
+
+// WhiteList define white list of data pool
+message WhiteList {
```

Comment:
It looks like a list from the name alone, but it actually contains only one piece of information.
Umm.. Naming is very hard....
Please see below for reference only
```proto
message WatingForTransmissionNFT {
  uint64 pool_id = 1;
  string address = 2;
}

message WhiteList {
  repeat WatingForTransmissionNFT  = 1;
}
```

Reply:
Well, no. I think this name is fine 😊

Reply:
Ah, got your point. This `WhiteList` was intended to be a single piece of white list. But in `GenesisState` above, there is an array of white lists which have all pieces of white lists. I should use plural expression of white list in `GenesisState`.

Reply:
In `GenesisState`, renamed it in [e5d8c20](https://github.com/medibloc/panacea-core/pull/306/commits/e5d8c2039353b0cbd51d364469ca6f37e5359ce7)
```proto
message GenesisState {
  ...
  repeated WhiteList     white_lists = 5 [(gogoproto.nullable) = false];
}
```